hooks/shapediver/viewer/drawing/useDrawingToolsEvents.ts

```diff
@@ -0,0 +1,93 @@
+import { addListener, EVENTTYPE_DRAWING_TOOLS, IEvent, removeListener } from "@shapediver/viewer";
+import { DrawingToolsEventResponseMapping, PointsData } from "@shapediver/viewer.features.drawing-tools";
+import { notifications } from "@mantine/notifications";
```

Comment:
@MajorMeerkatThe3rd we do have a `NotificationContext` for getting the notifications object, please use it instead